From 012f73e271638430e035ca68f5803dc2356d6a3e Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Tue, 30 Oct 2012 02:54:25 +0100
Subject: [PATCH 158/278] x86inc: Only define program_name if the macro is
 unset

This allows overriding the value from outside of the file.
---
 libavutil/x86/x86inc.asm |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libavutil/x86/x86inc.asm b/libavutil/x86/x86inc.asm
index 1fe9f55..b0df2b2 100644
--- a/libavutil/x86/x86inc.asm
+++ b/libavutil/x86/x86inc.asm
@@ -34,7 +34,9 @@
 ; as this feature might be useful for others as well.  Send patches or ideas
 ; to x264-devel@videolan.org .
 
-%define program_name ff
+%ifndef program_name
+    %define program_name ff
+%endif
 
 %define WIN64  0
 %define UNIX64 0
-- 
1.7.9.5

